src/main/java/com/app/onlinebookstore/controller/OrderController.java

```diff
@@ -0,0 +1,70 @@
+package com.app.onlinebookstore.controller;
+
+import com.app.onlinebookstore.dto.order.OrderCreateRequestDto;
+import com.app.onlinebookstore.dto.order.OrderDto;
+import com.app.onlinebookstore.dto.order.OrderItemDto;
+import com.app.onlinebookstore.dto.order.OrderUpdateRequestDto;
+import com.app.onlinebookstore.service.OrderService;
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import org.springframework.data.domain.Pageable;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PatchMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@Tag(name = "Order management",
+        description = "Endpoints for managing orders and orderItems")
+@RequiredArgsConstructor
+@RestController
+@RequestMapping(value = "/orders")
+public class OrderController {
+    private final OrderService orderService;
+
+    @PreAuthorize("hasRole('ROLE_USER')")
```

Comment:
check role "user" is redundant. Please, remove it everywhere

Reply:
about that someone talk to use it like in shoppingCart